### PR TITLE
[control-bar-test] clone to parse/stringify

### PR DESCRIPTION
**Pull request by KseniiaSokolovaEpam**
Way to clone skinConfig for controlBar-test was changed from using an utils function to JSON methods.

### CODE REVIEW

File: tests/components/controlBar-test.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ var _ = require('underscore');
 describe('ControlBar', function() {
 
   var baseMockController, baseMockProps;
-  var defaultSkinConfig = Utils.clone(skinConfig);
+  var defaultSkinConfig = JSON.parse(JSON.stringify(skinConfig));
```
Comment (jpfranco) [inline tests/components/controlBar-test.js]: Thanks for the update, can we also restore this for line 81?

**Comment by jpfranco** [comment]
Some context about why this is needed, for the second reviewer: https://github.com/ooyala/html5-skin/pull/1082/files#r197206826

**Comment by aeng7** [comment]
LGTM other than @jpfranco's comment

**Comment by jpfranco** [comment]
I've included these changes in https://github.com/ooyala/html5-skin/pull/1099, will close this.

**Pull request closed without merging**